styles.css

```
@@ -0,0 +1,4 @@
+button{
+  text-transform: capitalize !important;
+  font-size: 1rem !important;
+}
```

```suggestion
button {
  text-transform: capitalize !important;
}
```

I don't think it's  a good idea to set font-size into a fixed value as it will be update from vuetify based on width